### PR TITLE
[Doc] Modified docs in some files

### PR DESCRIPTION
**Pull request by Mr-DG-Wick**
Signed-off-by: Mr-DG-Wick <deepakgdkg1g8868@gmail.com>

**Description**
Changed files include FAQ ,Brand kit files and the career page files.
- Fixed typo's
- minor grammatical changes

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/assets/data/faq/index.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ const data = {
       category: "Meshery",
       answer: [
         "Meshery is an open source, vendor neutral projects that facilitates testing across meshes.",
-        "Other performance benchmark tools are not packaged into a mesh testing utility, butare only a load-generators unto their own.",
+        "Other performance benchmark tools are not packaged into a mesh testing utility, but are only a load-generators unto their own.",
```
Comment (Mr-DG-Wick) [inline src/assets/data/faq/index.js:177]: I'm not able to digest this part : `but are only a load-generators unto their own.`

**Comment by l5io** [comment]
🚀 Preview for commit 42947a04f1376e7af5fd93d8dcc296f77202928a at: https://618eb7844141ca0077987442--layer5ng.netlify.app

File: src/assets/data/faq/index.js [hunk 2]
```diff
@@ -216,8 +216,8 @@ const data = {
...
       category: "internships",
       answer: [
-        "Continuously. Layer5 has internships starting and stopping frequently. Some internships rigidly start and stop on particular dates, particularly those hosted in partnership with other organizations like the Linux Foundation, Google, MLH, Outreachy, and so on. Internships offered directly with Layer5 often start as a batch of candidantes show collective readiness.",
-        "We work with individuals to align with timing of internships with other activities in their life. build it into something active and something to be maintained.",
+        "Continuously. Layer5 has internships starting and stopping frequently. Some internships rigidly start and stop on particular dates, particularly those hosted in partnership with other organizations like the Linux Foundation, Google, MLH, Outreachy, and so on. Internships offered directly with Layer5 often start as a batch of candidates show collective readiness.",
+        "We work with individuals to balance out the timing of internships with other activities in their life. Build it into something active and something to be maintained.",
```
Comment (vedant-kakde) [inline src/assets/data/faq/index.js:220]:
```diff
- We work with individuals to align with timing of internships with other activities in their life.
+ We work with individuals to align with the timings of internships with other activities in their life.
```

**Review by vedant-kakde — CHANGES_REQUESTED** [review]
Thanks @Mr-DG-Wick
Some minor suggestions.

File: src/assets/data/faq/index.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ const data = {
       question: "As a member of one of Layer5's GitHub organizations, What is expected of me?",
       category: "Layer5",
       answer: [
-        "As a member of one of Layer5's GitHub organizations, your user account is entitled with triage level permissions across all repositories in the org. You are expected to engage where the topic strikes your interest, triaging (assigning, clarifying, labeling, qualifying) any issue you feel confortable doing so. You are encouraged to review any open pull request and offer feedback to or ask questions of the author.",
+        "As a member of one of Layer5's GitHub organizations, your user account is entitled with triage level permissions across all repositories in the org. You are expected to engage where the topic strikes your interest, triaging (assigning, clarifying, labeling, qualifying) any issue you feel confortable doing so. You are encouraged to review any open pull request and offer feedback to or ask questions to the author.",
```
[the remark this answers is not in the document]
Comment (Mr-DG-Wick) [reply src/assets/data/faq/index.js:16]: thanks for reviewing @vedant-kakde ! I'll change asap.

File: src/assets/data/faq/index.js [hunk 4]
```diff
@@ -31,15 +31,15 @@ const data = {
     {
       question: "I see works of other contributors being highlighted. Will my contributions be highlighted, too?",
       category: "Layer5",
-      answer: [ "We might not get to everyone, but we try to elevate the works of all of our contributors. All of our community members are proud of their work and so are we! We want their work and names to be recognized across our collective technology industry. Be sure to follow and engage with the Twitter, YouTube, and LinkedIn accounts."
+      answer: [ "We might not get to everyone, but we try to elevate the works of all of our contributors. All of our community members are proud of their work and so are we! We want their work and names to be recognized across our collective technology industry. Make sure that you follow and engage with the Twitter, YouTube, and LinkedIn accounts."
```
[the remark this answers is not in the document]
Comment (Mr-DG-Wick) [reply src/assets/data/faq/index.js:34]: what about : `Make sure you follow and engage with the Twitter, YouTube, and LinkedIn accounts.` 
@vedant-kakde

File: src/assets/data/faq/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (Mr-DG-Wick) [reply src/assets/data/faq/index.js]: done!

**Comment by l5io** [comment]
🚀 Preview for commit 37672d1afa4718759b1fdccff5e9e175389dd6fd at: https://618ec14c8971c81214a44321--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b3753b57de6fc3fd3859cdaf8d7ce5d3e795e757 at: https://619127dbcf0f0a3d132c61ae--layer5ng.netlify.app

File: src/sections/Company/Brand/Brand-components/servicemeshpatterns.js [hunk 5]
```diff
@@ -67,7 +67,7 @@ const ServiceMeshPatterns = () => {
         </Col>
         <Col>
           <p>
-                            The Service Mesh Pattern color palette consists of gray, green and colors and
+                            The Service Mesh Pattern color palette consists of gray, green and blue colors along with
```
Comment (warunicorn19) [inline src/sections/Company/Brand/Brand-components/servicemeshpatterns.js:70]: we don't use blue in service-mesh-patterns, just green and gray.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

File: src/sections/Company/Brand/Brand-components/servicemeshpatterns.js (hunk 5 above)
Comment (Mr-DG-Wick) [reply src/sections/Company/Brand/Brand-components/servicemeshpatterns.js:70]: Thanks for reviewing @warunicorn19 . So shall I frame it this way : `The Service Mesh Pattern color palette consists of gray and green colors along with .. ` ?

Comment (warunicorn19) [reply src/sections/Company/Brand/Brand-components/servicemeshpatterns.js:70]: Yup, sounds good👍

**Comment by l5io** [comment]
🚀 Preview for commit 2fa914184bc036aa92895535c8157a3481aa226c at: https://619343b8efbce136c5e1dcf7--layer5ng.netlify.app

File: src/assets/data/faq/index.js (hunk 1 above)
Comment (Mr-DG-Wick) [reply src/assets/data/faq/index.js:177]: @warunicorn19 Can you help me with this?

File: src/sections/Company/Brand/Brand-components/smp.js [hunk 6]
```diff
@@ -93,7 +93,7 @@ const SMPBrand = () => {
           </Col>
           <Col>
             <p>
-                            The SMP color palette consists of gray, green and colors and
+                            The SMP color palette consists of gray and green colors along with
```
Comment (warunicorn19) [inline src/sections/Company/Brand/Brand-components/smp.js:96]:
```suggestion
                            The SMP color palette consists of gray, green and blue colors along with
```

File: src/assets/data/faq/index.js (hunk 1 above)
Comment (warunicorn19) [reply src/assets/data/faq/index.js:177]: i don't see anything wrong with the line except for the `a` in there.

File: src/assets/data/faq/index.js [hunk 7]
```diff
@@ -174,7 +174,7 @@ const data = {
       category: "Meshery",
       answer: [
         "Meshery is an open source, vendor neutral projects that facilitates testing across meshes.",
```
Comment (warunicorn19) [inline src/assets/data/faq/index.js:176]:
```suggestion
        "Meshery is an open source, vendor neutral project that facilitates testing across meshes.",
```

File: src/assets/data/faq/index.js (hunk 1 above)
Comment (warunicorn19) [reply src/assets/data/faq/index.js:177]:
```suggestion
        "Other performance benchmark tools are not packaged into a mesh testing utility, but are only load-generators unto their own.", 
```

Comment (Mr-DG-Wick) [reply src/assets/data/faq/index.js:177]: ohh alright! I think my lack of knowledge on the topic.

Comment (warunicorn19) [reply src/assets/data/faq/index.js:177]:
```suggestion
        "Other performance benchmark tools are not packaged into a mesh testing utility, but are only load-generators unto their own.", 
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Review by warunicorn19 — APPROVED** [review]
Thank you @Mr-DG-Wick

**Pull request merged**